lib/spack/spack/test/cmd/env.py

```diff
@@ -665,117 +672,94 @@ def test_env_config_view_default(
 
 
 def test_env_updates_view_install_package(
-    tmpdir, mock_stage, mock_fetch, install_mockery
-):
+        tmpdir, mock_stage, mock_fetch, install_mockery):
     view_dir = tmpdir.mkdir('view')
     env('create', '--with-view=%s' % view_dir, 'test')
     with ev.read('test'):
         install('--fake', 'mpileaks')
 
-    assert os.path.exists(str(view_dir.join('.spack/mpileaks')))
-    # Check that dependencies got in too
-    assert os.path.exists(str(view_dir.join('.spack/libdwarf')))
+    check_mpileaks_install(view_dir)
 
 
 def test_env_updates_view_add_concretize(
-    tmpdir, mock_stage, mock_fetch, install_mockery
-):
+        tmpdir, mock_stage, mock_fetch, install_mockery):
     view_dir = tmpdir.mkdir('view')
     env('create', '--with-view=%s' % view_dir, 'test')
     install('--fake', 'mpileaks')
     with ev.read('test'):
         add('mpileaks')
         concretize()
 
-    assert os.path.exists(str(view_dir.join('.spack/mpileaks')))
-    # Check that dependencies got in too
-    assert os.path.exists(str(view_dir.join('.spack/libdwarf')))
+    check_mpileaks_install(view_dir)
 
 
 def test_env_updates_view_uninstall(
-    tmpdir, mock_stage, mock_fetch, install_mockery
-):
+        tmpdir, mock_stage, mock_fetch, install_mockery):
```

Comment:
(question) Was flake complaining about this before? If not, my general preference is to minimize style changes (no need to change this back though).

Reply:
No.  IIRC I was changing one or more and made them consistent with what I see in other test drivers.  But I will endeavor to refrain from style changes in the future.